### PR TITLE
Mh add glossary

### PR DESCRIPTION
**Pull request by bleehu**
I chose as the format to do Word:Definition on one line. That would make it really easy for CXDocs to pull the most up-to-date information and parse it straight from Git so that we could display it prettily without having to update CXDocs every time we add a definition. If we hate that idea, I can break it up much nicer.

### CODE REVIEW

File: Glossary_of_Terms.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Character:
+Character, Player: See Player Character
+Character, Non Player: See NPC
+
+Class: Most Player Characters have a Class. Classes are listed and explained in the CharacterCreation/02_Classes.txt document. 
+Class Feats: Every certain number of levels, your character gains an extra-special kind of feat depending on their chosen Class. These are listed in the CharacterCreation/02_Classes.txt document and are called Class Feats.
+
+D10: A 10 sided Dice. See The Primary Game Mechanic in the Basic Rules. 
+
+DM: Dungeon Master. See Galaxy Master.
```
Comment (Turtlelord26) [inline Glossary_of_Terms.txt]: No Galaxy Master entry.

File: Glossary_of_Terms.txt [hunk 2]
```diff
@@ -0,0 +1,37 @@
+Here are a list of some proper nouns and terms which are different in Compound X then in the English Language normally. If one is missing, please create an issue in Github asking for it! Hope it helps!
+
+Action: In Compound X, an Action rarely means a single abstract bodily movement as it does in English. Instead an Action is a measure of time specifically during a Player's turn in Combat. See Actions in Basic Rules.txt
```
Comment (Turtlelord26) [inline Glossary_of_Terms.txt]: is "Combat" going to be an entry? If not it shouldn't be capitalized.

**Review by Turtlelord26 — APPROVED** [review]
PR incoming for grammar.

File: Glossary_of_Terms.txt (hunk 1 above)
Comment (bleehu) [reply Glossary_of_Terms.txt]: Fixed.

File: Glossary_of_Terms.txt (hunk 2 above)
Comment (bleehu) [reply Glossary_of_Terms.txt]: Added

**Comment by bleehu** [comment]
Should I hold the merge for your grammar re-work?

**Comment by bleehu** [comment]
Awaiting merge of grammar fixes from Turtle's grammar check.

**Pull request merged**